Sources/IdentifiableProtocol+CellRegistration.swift

```diff
@@ -25,11 +25,19 @@ public extension UITableView {
     public func register(cellClass: UITableViewCell.Type) {
         self.registerClass(cellClass.self, forCellReuseIdentifier: cellClass.reuseIdentifier)
     }
+
+    public func registerNib(cellClass: UITableViewCell.Type) {
+        self.registerNib(UINib.init(nibName: String(cellClass), bundle: nil), forCellReuseIdentifier: cellClass.reuseIdentifier)
+    }
 }
 
 public extension UICollectionView {
     public func register(cellClass: UICollectionViewCell.Type) {
         self.registerClass(cellClass.self, forCellWithReuseIdentifier: cellClass.reuseIdentifier)
     }
+
+    public func registerNib(cellClass: UICollectionViewCell.Type) {
```

Comment:
I think we could wait until someone screams, adding it right now would mean that the autocompletion gets a bit messy since most of the time you want to use the main bundle.